src/theme.js

```diff
@@ -427,7 +427,7 @@ export const Mixins = {
     font-weight: 400;
   `,
   textLevel2: `
-    font-size: 2rem;
+    font-size: 1.5rem;
```

Comment:
It looks like this change has affected H2 size on regular desktop too.
![image](https://user-images.githubusercontent.com/40891631/106248949-9fca6000-6209-11eb-8341-9c184c4b58d7.png)

To clarify:
- on M and above, H2's should be 2rem (32px)
- below M, H2s should be 1.5rem (24px)